src/sentry/static/sentry/app/views/alerts/utils.tsx

```diff
@@ -145,3 +146,8 @@ export function getIncidentDiscoverUrl(opts: {
     ...toObject,
   };
 }
+
+export const DATA_SOURCE_LABELS = {
+  [Dataset.ERRORS]: t('Errors'),
+  [Dataset.TRANSACTIONS]: t('Transactions'),
```

Comment:
I wonder if this is used anywhere else? hm, maybe grep for `t('Transactions')`

Reply:
Oh yup seems like the choices for building an alert use the exact same text, I can change that over as well in a follow up PR

https://github.com/getsentry/sentry/blob/81cb17c6ad395dc914abb038d1134991de290496/src/sentry/static/sentry/app/views/settings/incidentRules/ruleConditionsForm.tsx#L124-L125

Reply:
sgtm!